README.md

```diff
@@ -80,6 +80,32 @@ load module in node.js style as `require('pica')` - your project MUST
 be compiled with [browserify](https://github.com/substack/node-browserify)
 to properly use Web Workers. In other case - use `require('pica/dist/pica')`.
 
+webpack:
+
+If you use Webpack to bundle your application, you must to define a resolve
```

Comment:
`must` is a bit too "strong". This looks like official and the only possible correct use.

Could you rephrase this, to make it look llike "notice" or "advice"?